meta-xt-prod-cockpit-rcar-driver-domain/recipes-kernel/linux/files/r8a7795-h3ulcb-4x2g-kf-doma.dtsi

```diff
@@ -8,6 +8,24 @@
 	#address-cells = <2>;
 	#size-cells = <2>;
 
+	cpu_scp_shm: scp-shmem@57000000/*0x1FF0000*/ {
+		compatible = "arm,scmi-shmem";
+		reg = <0x0 0x57000000 0x0 0x1000>;
+	};
+	scp-shmem@57001000 {
+		compatible = "arm,scmi-shmem";
+		reg = <0x00 0x57001000 0x00 0x1000>;
+	};
+	scp-shmem@57002000 {
+		compatible = "arm,scmi-shmem";
+		reg = <0x00 0x57002000 0x00 0x1000>;
+	};
+	scp-shmem@0x57003000 {
```

Comment:
Hi @iusyk, may we change it to scp-shmem@57003000 without '0x' to have a syntax similar to other added blocks?